### ISSUE TITLE
Invalid URL

### ISSUE THREAD

**Issue by r1de** [netdata/netdata#5479]
Forgive me if this has been asked before, I could not find it.

I have just setup streaming from my cloud server to my home network.
The cloud server is setup with memory mode = none web mode = none.
I am receiving metrics from the cloud just fine. I can select the other server from the "my-netdata" dropdown. But I cannot go back to view the metrics of the "main" server.
When I try, I get an "Invalid URL" pop up in my browser.
I've tried searching here and on google and I must not be using the correct terms, as I can't really seem to find any info.
Perhaps a graphic will explain better:

![netdata_invalidurl](https://user-images.githubusercontent.com/4380477/53281605-4d5c9c80-36e0-11e9-9271-9dbc6909b447.gif)

I have looked at the Firefox dev tools console and I see that when I click that link, the URL is:
http://hostname:19999/base/#menu_system_submenu_cpu;theme=slate;help=true

when it should just be:
http://hostname:19999/#menu_system_submenu_cpu;theme=slate;help=true

I am sure I missed something simple, but I just can't find it.
##### OS / Environment
"Main" server (master): Slackware 14.2
Cloud server (slave): Arch Linux
Browser: Firefox 60.4.0esr (64-bit) on Linux

##### Expected results
To be able to go back and view the metrics of the "main" server again.

**Comment by gmosx** [comment]
Hi,

Most probably you get a message like this in the console:

```
invalid URL detected:
...
```

Can you paste the exact URL here?

Also from the screenshot I see that you are using an older version of netdata. Could you try with the latest release?

**Comment by r1de** [comment]
URL is this:
```
invalid URL detected:
base/#menu_system_submenu_cpu;theme=slate;help=true
```
Your right, I am running 1.11.0...
attempting upgrade to 1.12, will report back.

Thanks!

**Comment by gmosx** [comment]
The URL is indeed invalid, there is no `http://` or `https://` prefix. Will have a look, but could you tell me the correct URLs for both servers?

**Comment by r1de** [comment]
Both these URL's come from right clicking and choosing "Copy Link Location"

I have upgraded to 1.12.1 and the issue persists.

This is the "main" (master) server link on the my-netdata dropdown:
`http://camphost:19999/base/#menu_system_submenu_cpu;theme=slate;help=true`
(if I remove "base/" from the above URL it works)

This is the other link on the dropdown for the streamed metrics from the cloud server:
`http://camphost:19999/host/cloud1/#menu_system_submenu_cpu;theme=slate;help=true`

Thank you for your help!

**Comment by BruceHowells** [comment]
Hate to add a "me too", but I was literally about to go looking at this problem today after standing up streaming yesterday.  Same experience and reproducer that @r1de reports.

**Comment by BruceHowells** [comment]
Actually, I can't reproduce this anymore on v1.12.1-7-g86ed386.

**Comment by r1de** [comment]
Now I will add the "me too".
I have since upgraded both of my netdata server to at least 1.12.1 and I no longer have the issue.

Thanks!

**Comment by cakrit** [comment]
I would say that this **may** have something to do with the fix in #5371, but I never saw corrupted links myself before that fix. I just didn't see the entries under 'Databases monitored by this host' appearing at all. I hadn't tested in Firefox, but I doubt it would have made a difference. It would be good to know what it is that fixed it, but we can close it too, since it's no longer there...
